internal/configs/sharding/localnet.go

```diff
@@ -22,7 +22,7 @@ const (
 	localnetVdfDifficulty  = 5000 // This takes about 10s to finish the vdf
 	localnetConsensusRatio = float64(0.1)
 
-	localnetFirstCrossLinkBlock = 13
+	localnetFirstCrossLinkBlock = 3
```

Comment:
does this exist because cross shard operation (transaction) takes "at most" 3 blocks generation to finalize?

Is cross link created from the receiver side? @chaosma @rlan35

Reply:
It is used to deal with fork condition, because we don't have crosslink before. it will only take effect after some block